### PR TITLE
integration test should fail if group_name is absent in email

### PR DESCRIPTION
**Pull request by alexkonkin**
Done - please review and confirm

### CODE REVIEW

**Comment by aaron-lane** [comment]
Fixes #105.

File: test/integration/minimal/controls/minimal.rb [hunk 1]
```diff
@@ -74,3 +76,16 @@
     end
   end
 end
+
+control 'group_email' do
+  title 'Group e-mail validation'
+
+  describe "e-mail" do
```
Comment (kopachevsky) [inline test/integration/minimal/controls/minimal.rb]: change to group_email

File: test/integration/minimal/controls/minimal.rb [hunk 2]
```diff
@@ -74,3 +76,16 @@
     end
   end
 end
+
+control 'group_email' do
```
Comment (kopachevsky) [inline test/integration/minimal/controls/minimal.rb]: do not create new control. just put new describe inside project-factory-minimal

Comment (kopachevsky) [inline test/integration/minimal/controls/minimal.rb]: should not create new **control** use existing _project-factory-minimal_

File: test/integration/minimal/controls/minimal.rb [hunk 3]
```diff
@@ -73,4 +75,11 @@
       expect(liens).to be_empty
     end
   end
+
+  describe "group_email" do
+    it "should be empty if user_group is empty" do
```
Comment (aaron-lane) [inline test/integration/minimal/controls/minimal.rb]:
```suggestion
    it "should be empty when group_name is empty" do
```

File: test/integration/minimal/controls/minimal.rb [hunk 4]
```diff
@@ -73,4 +75,11 @@
...
+      expect(group_email).to be_empty
+      expect(group_name).to be_empty
```
Comment (aaron-lane) [inline test/integration/minimal/controls/minimal.rb]: We should only have one assertion per example to simplify debugging when failures occur. Please move this to a separate `it` example.

File: test/integration/minimal/controls/minimal.rb (hunk 3 above)
Comment (alexkonkin) [reply test/integration/minimal/controls/minimal.rb]: Thank you for your review, done.

File: test/integration/minimal/controls/minimal.rb (hunk 4 above)
Comment (alexkonkin) [reply test/integration/minimal/controls/minimal.rb]: Thank you for your review. Done.

**Comment by alexkonkin** [comment]
I will update the CHANGELOG.md after your review.

File: test/integration/minimal/controls/minimal.rb [hunk 5]
```diff
@@ -73,4 +75,13 @@
       expect(liens).to be_empty
     end
   end
+
+  describe "group_email" do
+    it "should be empty when group_name is empty" do
```
Comment (aaron-lane) [inline test/integration/minimal/controls/minimal.rb]:
```suggestion
    it "group_name should be empty" do
```

**Comment by aaron-lane** [comment]
Thanks @alexkonkin!

**Pull request merged**